File: C#/Lv_0/마지막 두 원소.cs
using System;

public class Solution {
    public int[] solution(int[] num_list) {
        int[] a = new int[num_list.Length+1];
        for(int i=0; i<num_list.Length; i++) a[i] = num_list[i];
        if(a[a.Length-2] > a[a.Length-3]) a[a.Length-1] = a[a.Length-2]-a[a.Length-3];
        else a[a.Length-1] = a[a.Length-2]*2;
        return a;
    }
}

File: JavaScript/Lv_0/마지막 두 원소.js
function solution(num_list) {
    var a = num_list;
    a.at(-1) > a.at(-2) ? a.push(a.at(-1)-a.at(-2)) : a.push(a.at(-1)*2);
    return a;
}

// 더 깔끔하고 간단한 방법
// function solution(num_list) {
//     const [a, b] = [...num_list].reverse();
//     return [...num_list, a > b ? (a-b):a*2];
// }